ext/socket/raddrinfo.c

```diff
@@ -469,7 +469,7 @@ cancel_getaddrinfo(void *ptr)
     rb_nativethread_lock_unlock(&arg->lock);
 }
 
-static int
+int
 do_pthread_create(pthread_t *th, void *(*start_routine) (void *), void *arg)
```

Comment:
Can we add a prefix for non-static functions to avoid conflicting as much as possible?

Reply:
Thanks for pointing that out.I have remedied this with [27faf7f](https://github.com/ruby/ruby/pull/11653/commits/27faf7fcaeee92e0a664132ceb87843823301fec).

Reply:
Thanks. But this adds more non-prefixed functions...
For example: https://github.com/ruby/ruby/pull/11653/files#diff-443562a7b162dc0d001a053a745f194c4dbbae5a68d94e9d385d86d9e670431bR826